apps/native-component-list/src/screens/VideoThumbnailsScreen.tsx

```diff
@@ -0,0 +1,50 @@
+import * as VideoThumbnails from 'expo-video-thumbnails';
+import React, { useState } from 'react';
+import { StyleSheet, Button, View, Image, Text } from 'react-native';
+
+export default function VideoThumbnailsScreen() {
+  const [image, setImage] = useState<string | null>(null);
+
+  const generateThumbnail = async (sourceFilename: string) => {
+    try {
+      const { uri } = await VideoThumbnails.getThumbnailAsync(sourceFilename, {
+        time: 15000,
+      });
+      setImage(uri);
+    } catch (e) {
+      console.warn(e);
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      {image && <Image source={{ uri: image }} style={styles.image} />}
+      <Text>{image}</Text>
+      <Button
+        onPress={() => generateThumbnail('http://d23dyxeqlo5psv.cloudfront.net/big_buck_bunny.mp4')}
```

Comment:
Afaik iOS by default is not allowing insecure HTTP connections, so let's use HTTPS
```suggestion
        onPress={() => generateThumbnail('https://d23dyxeqlo5psv.cloudfront.net/big_buck_bunny.mp4')}
```